src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-row-view.ts

```diff
@@ -317,27 +317,39 @@ class GmailThreadRowView {
     this._userView = userView;
   }
 
-  getCounts(): Counts {
-    let counts = this._counts;
-
-    if (!counts) {
-      const recipientsElement = querySelector(this._elements[0], 'td div.yW');
```

Comment:
Wrapping this in a try catch and then using the SDK logger might be a more visible way to report selector issues.